packages/blade/src/components/Breadcrumb/types.ts

```diff
@@ -0,0 +1,64 @@
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import type { IconComponent } from '~components/Icons';
+import type { LinkProps } from '~components/Link';
+
+type BreadcrumbProps = StyledPropsBlade & {
+  /**
+   * Size of the Breadcrumb
+   *
+   * @default medium
+   */
+  size?: 'small' | 'medium' | 'large';
+  /**
+   * Color of the Breadcrumb
+   *
+   * @default neutral
+   */
+  color?: 'neutral' | 'primary' | 'white';
+  /**
+   * Content of the Breadcrumb, accepts BreadcrumbItem
+   */
+  children: React.ReactNode;
+  /**
+   * Whether to show the last separator
+   */
+  showLastSeparator?: boolean;
+  /**
+   * aria-label for breadcrumb
+   */
+  accessibilityLabel?: string;
+};
+
+type BreadcrumbItemProps = {
+  /**
+   * Href of the BreadcrumbItem
+   */
+  href: string;
```

Comment:
Should we also add `target`?

Reply:
I don't think that should be needed. 

A breadcrumb shows a user journey, so it's unlikely that you will by default want to open a breadcrumb link on a new tab, if user needs to open a link on a new tab they can `ctrl + click` the link